cypress/e2e/components/filings/section.cy.ts

```diff
@@ -227,7 +227,7 @@ context('Filings history section', () => {
     ).as('officerChangeDocumentList')
 
     // expand filing
-    cy.get(`[data-cy="filingHistoryItem-default-filing-${officerChange.filingId}"]`)
+    cy.get(`[data-cy="filingHistoryItem-default-filing-${officerChange.filingId}"]`, { timeout: 10000 })
```

Comment:
My changes have nothing to do with this pendingOfficerChange/officerChange test, but this test has always been just on the edge of failing due to long delays in getting the data or click actions. Adding this timeout should fix it.

(Probably if I'd restarted this check step, it would have worked the secod time.)